Error: Cannot find module 'q'

Hi,

The package is throwing an error when used because `q` is listed as a _development_ dependency in `package.json` instead of a normal one. Here is the stack trace:

```
module.js:338
    throw err;
          ^
Error: Cannot find module 'q'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/Razvan/Workspace/sso-app/node_modules/gulp-scss/lib/scss.js:14:7)
    at Object.<anonymous> (/Users/Razvan/Workspace/sso-app/node_modules/gulp-scss/lib/scss.js:93:4)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
```

And here are the `devDependencies` and `dependencies` keys from your `package.json`:

```
  "devDependencies": {
    "chai": "^2.3.0",
    "coffee-script": "^1.9.2",
    "gulp": "^3.8.11",
    "gulp-coffee": "^2.3.1",
    "gulp-coffeelint": "^0.4.0",
    "gulp-mocha": "^2.0.1",
    "gulp-notify": "^2.2.0",
    "gulp-plumber": "^1.0.0",
    "gulp-sourcemaps": "^1.5.2",
    "gulp-util": "^3.0.4",
    "mocha": "^2.2.4",
    "q": "^1.3.0",
    "sinon": "^1.14.1"
  },
  "dependencies": {
    "convert-source-map": "^1.1.0",
    "deepcopy": "^0.5.0",
    "merge": "^1.2.0",
    "mkdirp": "^0.5.0",
    "node-kissdi": "^1.0.3",
    "through2": "^0.6.5",
    "vinyl-sourcemaps-apply": "^0.1.4"
  },
```

Thanks for taking a look at this.

--- a/package.json
+++ b/package.json
@@ -33,11 +33,11 @@
     "gulp-sourcemaps": "^1.5.2",
     "gulp-util": "^3.0.4",
     "mocha": "^2.2.4",
-    "q": "^1.3.0",
     "sinon": "^1.14.1"
   },
   "dependencies": {
     "convert-source-map": "^1.1.0",
+    "q": "^1.3.0",
     "deepcopy": "^0.5.0",
     "merge": "^1.2.0",
     "mkdirp": "^0.5.0",

